MaterialsLib/Adsorption/ReactionCaOH2.h

```diff
@@ -0,0 +1,90 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include "Reaction.h"
+#include "Adsorption.h"
+
+namespace ProcessLib
+{
+template<typename>
+class TESFEMReactionAdaptorCaOH2;
+}
+
+namespace Adsorption
+{
+
+class ReactionCaOH2 final : public Reaction
+{
+public:
+    explicit ReactionCaOH2(BaseLib::ConfigTree const& conf)
+        : ode_solver_config{conf.getConfSubtree("ode_solver_config")}
+    {
+        /*auto const param = conf.get_optional<double>("reaction_enthalpy");
+        if (param) {
+            _enthalpy = *param;
+        } else {
+            ERR("<reaction_enthalpy> not specified.");
+            std::abort();
+        }*/
```

Comment:
Is the commented code needed in the future? Could you add a comment to the code or remove it?